src/ni/vsbuild/steps/LvUnitTestFrameworkStep.groovy

```diff
@@ -0,0 +1,17 @@
+package ni.vsbuild.steps
+
+import ni.vsbuild.BuildConfiguration
+
+class LvUnitTestFrameworkStep extends LvProjectStep {
+
+   def UTF_path
```

Comment:
`def utfPath` is the variable naming convention for groovy.

Reply:
Removed UTF